.git-hooks/pre-commit

```diff
@@ -2,7 +2,7 @@
 #
 files_to_check="docs/src/recipes/*.ipynb"
 files_to_check+=" docs/src/user_guide.ipynb"
-files_to_check+=" docs/src/tutorials/getting_started.ipynb"
```

Comment:
Do we risk breaking existing url that we shared?

Maybe we can add a redirection from tutorials/getting_started.ipynb to getting_started.ipynb?

Reply:
Good idea, I'll create a symlink and see if it works.

Reply:
FYI: symlink didn't work, but I could configure a redirect through the rtd admin panel.